_activities/pyhep.md

```diff
@@ -0,0 +1,46 @@
+---
+title: PyHEP - "Python in HEP" Series of Workshops
+author: "Eduardo Rodrigues"
+layout: default
+redirect_from:
+  - /pyhep.html
+---
+
+# PyHEP "Python in HEP" Series of Workshops
+
+The **PyHEP workshops** are a new series of workshops initiated and supported by the HSF
+with the aim to provide an environment to discuss and promote the usage of Python in the HEP community at large.
+
+
+## PyHEP 2018
+
+The first workshop, **PyHEP 2018**, will be held as a pre-CHEP event in Sofia, Bulgaria, the weekend just before the start of the [CHEP 2018 conference](http://chep2018.org/), i.e. on July 7-8 2018. It will focus on the review of where and how Python is used in our community. The workshop will also be a forum for the participants, representatives of the community, to discuss topics around the areas of work identified in the HSF [Community White Paper](/activities/cwp.html). There will be ample time for discussion.
```

Comment:
I would take out the "i.e." and phrase as:

...will be held as a pre-CHEP event in Sofia, Bulgaria, on 7-8 July 2018, just before the start of the [CHEP 2018 conference](http://chep2018.org/).

"the review" -> "a review"

"our community." -> "our community and what the future will hold."

"will also" -> "will"

Reply:
Better indeed. All changed.